engines/gob/detection/tables_inca2.h

```diff
@@ -173,7 +173,7 @@
 {
 	{
 		"inca2",
-		"",
+		"v1.000",
```

Comment:
This part is added to the game description and presented to user.
As it has no value for most of our users, it's better to not confuse them.

Reply:
> This part is added to the game description and presented to user. As it has no value for most of our users, it's better to not confuse them.

Okay then we exclude it from the PR if this can confuse the user

Reply:
I agree with @lephilousophe on this. As we have discussed in the past, the version for a game entry is only useful when there were multiple versions released (e.g. there was a patch by the original vendor that fixed issues, and it would be useful for users to check if they have installed necessary patches). As far as we know, there were no patches released for Inca 2, so the version information isn't needed